svi/auto_thermal_reformer/condition_num_plots.py

```diff
@@ -0,0 +1,194 @@
+#  ___________________________________________________________________________
```

Comment:
I believe this script is superseded by `generate_iterate_data.py` and `plot_iterate_condition_numbers.py`.